case_studies/kubernetes/test_same.py

```diff
@@ -50,10 +54,10 @@ def canonicalize(doc):
 
 for filename in files:
     with open(filename + '.old.yaml', 'r') as f:
-        yaml_doc = canonicalize(yaml.load(f))
+        yaml_doc = canonicalize(yaml.load(f, Loader=yaml.FullLoader))
```

Comment:
I think it's better to use SafeLoader unless we have a good reason not to (in this case we're working with trusted data, so it shouldn't really matter, but let's stick with a better default).

Reply:
Sure, done!